pyrolite/util/classification.py

```diff
@@ -400,6 +400,46 @@ def __init__(self, **kwargs):
         super().__init__(**poly_config)
 
 
+class QAP(PolygonClassifier):
+    """
+    IUGS QAP ternary classification
+    [#ref_1]_ [#ref_2]_.
+
+    Parameters
+    -----------
+    name : :class:`str`
+        A name for the classifier model.
+    axes : :class:`list` | :class:`tuple`
+        Names of the axes corresponding to the polygon coordinates.
+    fields : :class:`dict`
+        Dictionary describing indiviudal polygons, with identifiers as keys and
+        dictionaries containing 'name' and 'fields' items.
+
+    References
+    -----------
+}    .. [#ref_1] Streckeisen, A. Classification and nomenclature of plutonic rocks
```

Comment:
There is a leading `}` which snuck in here and is breaking a few things. Most should be fine when this is removed.